widgets/tabs/tpl/default.php

```diff
@@ -27,7 +28,7 @@
 	<div class="sow-tabs-panel-container">
 	<?php foreach ( $tabs as $i => $tab ) : ?>
 		<div class="sow-tabs-panel">
-			<div class="sow-tabs-panel-content">
+			<div class="sow-tabs-panel-content" role="tabpanel" <?php echo $i != $initial_tab_index ? 'aria-hidden="true"' : ''; ?>>
```

Comment:
Doesn't this also need `tabindex=0` when selected? Or perhaps the parent container `sow-tabs-panel-container`?